cmd/tailscaled/depaware.txt

```diff
@@ -139,10 +139,10 @@ tailscale.com/cmd/tailscaled dependencies: (generated by github.com/tailscale/de
         bufio                                                        from compress/flate+
         bytes                                                        from bufio+
         compress/flate                                               from compress/gzip+
-        compress/gzip                                                from internal/profile+
+        compress/gzip                                                from net/http+
```

Comment:
I wonder if we're using different versions of Go to generate this.

Reply:
I'm using `./tool/go run github.com/tailscale/depaware -update tailscale.com/cmd/tailscaled`. Missing sort somewhere? Not sure why the DAG seems to have changed on some of these deps, although it did correctly record the movement of unsafe from router to netns.

Reply:
Did another refresh, and now depaware.txt doesn't make sense (new deps appeared that I haven't touched), and also the CI check fails. Please advise?

Reply:
Which version of Go? I'm using Go 1.15 on Linux. I believe the CI is too?

Reply:
My system go is 1.14.1, although I ran the update with `tool/go`, which should use our pinned toolchain... Shouldn't it?...

Reply:
Well, depaware itself shells out to the "go" tool, so I imagine `tool/go` isn't setting the $PATH/$GOROOT such that depaware when run then finds our Go as the `"go"` that it runs via `os/exec`.

If we stopped shelling out to go (as we'd want to do for https://github.com/tailscale/depaware/issues/2) then that's another way to solve it probably.

/cc @josharian

Reply:
Twiddled my $PATH and depaware makes sense now.

Reply:
Hmm, but CI depaware still fails?... Now I'm using our toolchain go for all uses of `go`. Further ideas welcome, otherewise I plan to ignore the CI failure in this PR and separately make it report a more useful error.